README.md

```diff
@@ -29,7 +29,7 @@ Benchmark (Mean time for 500 Kb payload, 2000 cycles, 1000 cycle warmup):
 
 | Library               | Mean time in nanoseconds (less is better) |
 |-----------------------|-------------------------------------------|
-| busboy 0.31           | 340114                                    |
+| busboy 0.3.1           | 340114                                    |
```

Comment:
Did you evaluate adding a new `version` column?

Reply:
no, the version of busboy is 0.3.1
https://www.npmjs.com/package/busboy

Reply:
@Uzlopak What @Eomm means, it would look better if we made it into three-column table where version would be a separate column. Which it probably would.